chat container crash

```
> chatapi@0.0.7 start /
> ts-node src/index.ts

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './_shims/auto/runtime' is not defined by "exports" in /node_modules/openai/package.json
    at applyExports (internal/modules/cjs/loader.js:498:9)
    at trySelf (internal/modules/cjs/loader.js:417:23)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1007:22)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (internal/modules/cjs/loader.js:899:27)
    at Module.require (internal/modules/cjs/loader.js:1090:19)
    at require (internal/modules/cjs/helpers.js:75:18)
    at Object.<anonymous> (/node_modules/openai/_shims/index.js:5:14)
    at Module._compile (internal/modules/cjs/loader.js:1201:30)
    at Module._extensions..js (internal/modules/cjs/loader.js:1221:10) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! chatapi@0.0.7 start: `ts-node src/index.ts`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the chatapi@0.0.7 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2024-03-19T14_16_41_843Z-debug.log

> chatapi@0.0.7 start /
> ts-node src/index.ts

Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './_shims/auto/runtime' is not defined by "exports" in /node_modules/openai/package.json
    at applyExports (internal/modules/cjs/loader.js:498:9)
    at trySelf (internal/modules/cjs/loader.js:417:23)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:1007:22)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (internal/modules/cjs/loader.js:899:27)
    at Module.require (internal/modules/cjs/loader.js:1090:19)
    at require (internal/modules/cjs/helpers.js:75:18)
    at Object.<anonymous> (/node_modules/openai/_shims/index.js:5:14)
    at Module._compile (internal/modules/cjs/loader.js:1201:30)
    at Module._extensions..js (internal/modules/cjs/loader.js:1221:10) {
  code: 'ERR_PACKAGE_PATH_NOT_EXPORTED'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! chatapi@0.0.7 start: `ts-node src/index.ts`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the chatapi@0.0.7 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2024-03-19T14_19_54_974Z-debug.log

```

--- a/docker/chatapi/amd64-Dockerfile
+++ b/docker/chatapi/amd64-Dockerfile
@@ -1,4 +1,4 @@
-FROM treehouses/node
+FROM treehouses/node:3.16
 
 RUN apk update
 

--- a/docker/chatapi/arm-Dockerfile
+++ b/docker/chatapi/arm-Dockerfile
@@ -6,7 +6,7 @@ RUN bash ./crosscompile_chatapi.sh armv7 install
 
 #####
 
-FROM treehouses/node-tags:arm
+FROM treehouses/node-tags:arm-3.16
 
 RUN apk update
 

--- a/docker/chatapi/arm64-Dockerfile
+++ b/docker/chatapi/arm64-Dockerfile
@@ -6,7 +6,7 @@ RUN bash ./crosscompile_chatapi.sh armv8 install
 
 #####
 
-FROM treehouses/node-tags:arm64
+FROM treehouses/node-tags:arm64-3.16
 
 RUN apk update
 

--- a/package.json
+++ b/package.json
@@ -1,9 +1,9 @@
 {
   "name": "planet",
   "license": "AGPL-3.0",
-  "version": "0.14.18",
+  "version": "0.14.21",
   "myplanet": {
-    "latest": "v0.14.21",
+    "latest": "v0.14.49",
     "min": "v0.13.69"
   },
   "scripts": {

--- a/src/app/chat/chat-window/chat-window.component.html
+++ b/src/app/chat/chat-window/chat-window.component.html
@@ -13,7 +13,6 @@
       <mat-label i18n>Chat</mat-label>
       <div class="textarea-container">
         <textarea matInput [formControl]="promptForm.controls.prompt" rows="5" placeholder="Send a message" i18n-placeholder></textarea>
-        <mat-error><planet-form-error-messages [control]="promptForm.controls.prompt"></planet-form-error-messages></mat-error>
         <button mat-icon-button [planetSubmit]="promptForm.valid && spinnerOn" [disabled]="!promptForm.valid" color="primary"><mat-icon>send</mat-icon></button>
       </div>
     </mat-form-field>

